src/main/java/com/synopsys/integration/detect/workflow/blackduck/project/SyncProjectOperation.java

```diff
@@ -43,7 +44,17 @@ public ProjectVersionWrapper sync(
             projectSyncOptions
         );
         boolean forceUpdate = projectSyncOptions.isForceProjectVersionUpdate();
-        return projectService.syncProjectAndVersion(projectSyncModel, forceUpdate);
+
+        //TODO- remove try-catch once there is a mechanism for validating property values against BD versions
+        try {
+            return projectService.syncProjectAndVersion(projectSyncModel, forceUpdate);
+        } catch (BlackDuckApiException e) {
+            if (projectSyncOptions.getCloneCategories().contains(ProjectCloneCategoriesType.DEEP_LICENSE)) {
+                String message = "Attempt to create or update project failed.  If you are using a Black Duck earlier than 2022.2.0, this may be because you passed DEEP_LICENSE as a project clone category, either explicitly or by passing ALL.";
+                throw new IntegrationException(message);
```

Comment:
Can we also throw the underlying exception? I don't want to lose it.

Reply:
@taikuukaits How do you mean?  I don't think it's possible to throw two exceptions.

Reply:
`IntegrationException` allows you to attach the original `Throwable` like `throw new IntegrationException(message, e)`

Reply:
Ah right, makes sense.